src/modals/Transfer.tsx

```diff
@@ -123,8 +115,7 @@ function Transfer(props: TransferProps) {
                 tokenAddress: state.token!.address,
                 tokenAmount: Number(fees),
                 transactionDetails: {
-                    retries: 7,
-                    ignoreTransactionReceipt: true
```

Comment:
Why did we decide to wait for the transaction receipt again?

Reply:
updated